smarctlide-monitoring/src/main/java/eu/smartclide/contexthandling/ServiceMain.java

```diff
@@ -0,0 +1,132 @@
+package eu.smartclide.contexthandling;
+/*-
+ * #%L
+ * smarctlide-monitoring
+ * %%
+ * Copyright (C) 2017 - 2019 ATB
+ * %%
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ * #L%
+ */
+
+import de.atb.context.monitoring.models.IMonitoringDataModel;
+import de.atb.context.services.AmIMonitoringService;
+import de.atb.context.services.IAmIMonitoringDataRepositoryService;
+import de.atb.context.services.IAmIMonitoringService;
+import de.atb.context.services.faults.ContextFault;
+import de.atb.context.services.wrapper.AmIMonitoringDataRepositoryServiceWrapper;
+import org.apache.cxf.endpoint.Server;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import de.atb.context.services.SWServiceContainer;
+import de.atb.context.services.manager.ServiceManager;
+import de.atb.context.tools.ontology.AmIMonitoringConfiguration;
+import org.springframework.util.ResourceUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Objects;
+import java.util.Properties;
+
+@SpringBootApplication
+public class ServiceMain {
+    private static final Logger logger = LoggerFactory.getLogger(ServiceMain.class);
+    private static Server server;
+    private static IAmIMonitoringService service;
+    private static AmIMonitoringDataRepositoryServiceWrapper monitoringDataRepository;
+    private static IAmIMonitoringDataRepositoryService<IMonitoringDataModel<?, ?>> reposService;
+
+    private static void initialize() {
+        Properties props = System.getProperties();
+        props.setProperty("org.apache.cxf.stax.allowInsecureParser", "true");
+
+        org.apache.log4j.BasicConfigurator.configure();
+
+        String smartclideConfigPath = "resources";
+
+        // Environment Variable
+        String smartclideHome = System.getenv("SMARTCLIDE_HOME");
+        if (smartclideHome != null && Files.exists(Paths.get(smartclideHome))) {
+            smartclideConfigPath = smartclideHome + File.separator + "config";
+        // Linux config directory /var/lib/smartclide
+        } else if (Files.exists(Paths.get("/var/lib/smartclide"))) {
+            smartclideHome = "/var/lib/smartclide";
+            smartclideConfigPath = smartclideHome + File.separator + "config";
+        // Linux config directory /opt/smartclide/config
+        } else if (Files.exists(Paths.get("/opt/smartclide"))) {
+            smartclideHome = "/opt/smartclide";
+            smartclideConfigPath = smartclideHome + File.separator + "config";
+        // Windows COnfigu Directories
```

Comment:
typo

Reply:
done

Reply:
I don't see any change. Did you maybe forget to push a commit?

Reply:
was already comitted and pushed